modal-styles.css

```diff
@@ -0,0 +1,100 @@
+.modal {
```

Comment:
- [ ] Please in some sizes the close button disappears fix this and make sure it is obvious in all screen sizes note that the button in the image is a part of the background it is not the close button.
![image](https://user-images.githubusercontent.com/74744787/217951375-6ee16ec8-602b-494e-a6ce-26f36c9b7bd6.png)